docker-compose.dev.yml

```diff
@@ -150,3 +150,14 @@ services:
     restart: "no"
   berichtencentrum-melding:
     restart: "no"
+  lmb-ldes-consumer:
+    environment:
+      CRON_PATTERN: "*/15 * * * * *" # run the job every 15 seconds
+    networks:
```

Comment:
can this be removed?

Reply:
Sure, could add it to the overwrite if needed